src/canvas.class.js

```diff
@@ -425,8 +425,12 @@
       if (this.hasLostContext) {
         this.renderTopLayer(this.contextTop);
       }
+
       var canvasToDrawOn = this.contextContainer;
-      this.renderCanvas(canvasToDrawOn, this._chooseObjectsToRender());
+      if (canvasToDrawOn) {
+        this.renderCanvas(canvasToDrawOn, this._chooseObjectsToRender());
+      }
```

Comment:
Fails kind of silently, but not sure what else to do?

Reply:
It does but as far as I've been able to tell, it seems to be happening when the user is navigating away so not much harm would come from it.